    @Test
    public void test4() throws Exception {
        helper2("fMe", "fYou", true, false, true, true, true, true);
    }

